app/components/Comments/CommentView.tsx

```diff
@@ -27,23 +51,87 @@ const CommentView = (props: Props) => {
     contentTarget,
     style,
     displayTitle = true,
-    newOnTop = false,
     contentAuthors,
   } = props;
   const commentFormProps = {
     contentTarget,
   };
-  const tree = generateTreeStructure(comments);
+  const [ordering, setOrdering] = useState<Option>(orderingOptions[0]);
+  const [sortedComments, setSortedComments] = useState<Comment[]>(comments);
+  const [displaySorting, setDisplaySorting] = useState(false);
```

Comment:
Is this state needed? Shouldn't this logic be handled by the Dropdown component?

Reply:
I don't think it is?